src/Common/src/Interop/Windows/Interop.HRESULT.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    // https://msdn.microsoft.com/en-us/library/cc231198.aspx
+    internal enum HRESULT : int
+    {
+        S_OK = 0,
```

Comment:
Nit, some of these are also defined in Gdiplus.cs

Reply:
Yeah, plan to get them all cleaned up.